publish-locally.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if [[ -z "$1" ]]
+  then
+    echo "No version provided"
+    exit 1
+fi
+
+
+yarn run bump "$1"
+
+yarn install
+source ./export_development_aliases.sh
```

Comment:
Does this line do anything?

Reply:
oh I guess it exports $OPTIC_SRC_DIR for below?

Reply:
Yes